onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -491,7 +490,7 @@ def collect_for_entropy(self, name_to_arr):
                 old_histogram = self.histogram_dict[tensor]
                 self.histogram_dict[tensor] = self.merge_histogram(old_histogram, data_arr, min_value, max_value, threshold)
             else:
-                hist, hist_edges = np.histogram(data_arr, self.num_quantized_bins, range=(-threshold, threshold))
+                hist, hist_edges = np.histogram(data_arr, 8192, range=(-threshold, threshold))
```

Comment:
Please don't hardcode using 8192.
We are going to make 'num_bins' and 'num_quantized_bins' be configurable, by doing so you can set num_bins=8192 and num_quantized_bin=255 or other values that can achieve better accuracy for your model. Please expect our OR in a few days.